### PR TITLE
Javalab: add neighborhood

### PR DESCRIPTION
**Pull request by molly-moen**
Incorporate the initial version of the neighborhood skin into Javalab. This PR adds those sprite mapping objects to the neighborhood skin configuration, updates Javalab to use the neighborhood skin, and switches to use version 2.1.0 of Maze.

Now level `Allthethings Javalab 1` (`/s/allthethings/lessons/44/levels/1`) looks like this (off by one pixel issues on sprites to be fixed later):
Screen Shot 2021-05-17 at 1 53 54 PM

## Links

- jira ticket: [CSA 292](https://codedotorg.atlassian.net/browse/CSA-292)

## Testing story
Tested locally. Most of the testing for this came with updating the Maze repo earlier. 

## Follow-up work
- update `neighborhood.js` in this repo to exercise Maze functionality
- add any required updates to the Maze repo for painting and painter movement.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/maze/neighborhoodSprites.js [hunk 1]
```diff
@@ -0,0 +1,1792 @@
+export const sheetRows = {
```
Comment (jmkulwik) [inline apps/src/maze/neighborhoodSprites.js]: Out of curiosity, why make this a js file instead of a json file?

Comment (molly-moen) [reply apps/src/maze/neighborhoodSprites.js]: We need to send it as a js object to maze, so this removes the step of converting it from json to js in code

Comment (jmkulwik) [reply apps/src/maze/neighborhoodSprites.js]: If you want to keep it as json, I think you can import it to js by default in this way: https://github.com/code-dot-org/code-dot-org/blob/30655d5cee703d6d5e0ae7d3265b554bf656b0ac/apps/test/unit/StudioAppTest.js#L21

**Review by jmkulwik — APPROVED** [review]
LGTM!

File: apps/src/maze/neighborhoodSprites.js (hunk 1 above)
Comment (molly-moen) [reply apps/src/maze/neighborhoodSprites.js]: oh cool, will update it to do that

**Pull request merged**